imod/mf6/gwtgwt.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional
+
+import cftime
+import numpy as np
+import xarray as xr
+
+from imod.mf6.auxiliary_variables import expand_transient_auxiliary_variables
+from imod.mf6.exchangebase import ExchangeBase
+from imod.mf6.package import Package
+from imod.typing import GridDataArray
+
+
+class GWTGWT(ExchangeBase):
+    """
+    This package is for writing an exchange file, used for splitting up a model
+    into different submodels (that can be solved in parallel). It (usually)
+    is not instantiated by users, but created by the "split" method of the
+    simulation class."""
```

Comment:
Please add a newline here:

```suggestion
    simulation class.
    """
```